examples/functions/functions.metatags

```diff
@@ -0,0 +1,2 @@
+description: Functions in Ballerina operate in the same way as in any other language. It is a mechanism to create a reusable unit of functionality within a program.
```

Comment:
This description needs to be updated. See the updated description in the corresponding BBE.

Reply:
What can we do for now?

Reply:
Shall we delete it and add when we have the proper description?

Reply:
I think we can just take the description from https://github.com/ballerina-platform/ballerina-distribution/pull/621/files right?

Reply:
Oh right, edited!